experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/FullscreenImage.kt

```diff
@@ -0,0 +1,222 @@
+package example.imageviewer.view
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsHoveredAsState
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowLeft
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.input.key.*
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.unit.IntRect
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.dp
+import example.imageviewer.Localization
+import example.imageviewer.core.BitmapFilter
+import example.imageviewer.core.FilterType
+import example.imageviewer.model.*
+import example.imageviewer.style.*
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.orEmpty
+import org.jetbrains.compose.resources.rememberImageBitmap
+import org.jetbrains.compose.resources.resource
+
+@Composable
+internal fun FullscreenImage(
+    picture: Picture?,
+    getImage: suspend (Picture) -> ImageBitmap,
+    getFilter: (FilterType) -> BitmapFilter,
+    localization: Localization,
+    back: () -> Unit,
+    nextImage: () -> Unit,
+    previousImage: () -> Unit,
+) {
+    val filtersState = remember { mutableStateOf(emptySet<FilterType>()) }
+
+    val originalImageState = remember(picture) { mutableStateOf<ImageBitmap?>(null) }
+    LaunchedEffect(picture) {
+        if (picture != null) {
+            originalImageState.value = getImage(picture)
+        }
+    }
+
+    val originalImage = originalImageState.value
+    val filters = filtersState.value
+    val imageWithFilter = remember(originalImage, filters) {
+        if (originalImage != null) {
+            var result: ImageBitmap = originalImage
+            for (filter in filters.map { getFilter(it) }) {
+                result = filter.apply(result)
+            }
+            result
+        } else {
+            null
+        }
+    }
+
+    Box(Modifier.fillMaxSize()) {
+        Column {
+            Toolbar(picture?.name ?: "", filtersState, localization, back)
+            if (imageWithFilter != null) {
+                val imageSize = IntSize(imageWithFilter.width, imageWithFilter.height)
+                val scalableState = remember(imageSize) { mutableStateOf(ScalableState(imageSize)) }
+                val visiblePartOfImage: IntRect = scalableState.value.visiblePart
+                Slider(
```

Comment:
The slider and Back/Forward colors don't fit into imageviewer theme:
![image](https://user-images.githubusercontent.com/5963351/209154669-b865f1d6-8630-428f-a0b3-c5f606530dfe.png)

Let's change them to something else (more grayish probably, not sure)

Reply:
Done

<img width="799" alt="image" src="https://user-images.githubusercontent.com/99798741/209206170-e2517fb0-55df-4382-b208-fcfdfaa102d0.png">